interfaces/mount/ns.go

```diff
@@ -38,7 +39,7 @@ func mountNsPath(snapName string) string {
 func runNamespaceTool(toolName, snapName string) ([]byte, error) {
 	mntFile := mountNsPath(snapName)
 	if osutil.FileExists(mntFile) {
-		toolPath := filepath.Join(dirs.DistroLibExecDir, toolName)
+		toolPath := cmd.InternalToolPath(toolName)
```

Comment:
Feels like a slightly odd place `cmd.InternalToolPath` but then it also fits - it is about commands afterall.